icons/xilinx.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Xilinx icon</title><path d="M23.7,0l-5.2,6l-5.2-6H23.7z M18.5,18l-5.2,6h10.4L18.5,18z M13.3,12L8,6l5.2-6H5.6L0.3,6l5.2,6l-5.2,6 l5.2,6h7.7L8,18L13.3,12z" /></svg>
```

Comment:
Hi @sharmavins23, thanks for your work on this!
I'm still seeing some slight differences between your version (red) and source (black). They have an SVG in the header of their website, which is what I've used for my source - may be worth rebuilding to that?
![image](https://user-images.githubusercontent.com/1830867/103439919-1c811300-4c39-11eb-9e6d-858eb5e505ab.png)

I've included my SVG below, which passes the linter, should you want to use that one:
```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Xilinx icon</title><path d="M8 18l5.241 6H5.586L.345 18l5.241-6L.345 6l5.241-6h7.655L8 6l5.241 6L8 18zM23.655 0H13.241l5.241 6 5.173-6zM13.241 24h10.414l-5.172-6-5.242 6z"/></svg>
```

I can also see a statement regarding their buyout by AMD - @simple-icons/maintainers should we still look to include this, on the assumption that AMD keep the brand going?